apps/src/code-studio/pd/fit_weekend_registration/TravelPlans.jsx

```diff
@@ -98,7 +94,7 @@ export default class TravelPlans extends LabeledFormComponent {
           {this.checkBoxesFor("dietaryNeeds")}
           {
             this.props.data.dietaryNeeds &&
-            this.props.data.dietaryNeeds.includes('Food Allergy') &&
+            (this.props.data.dietaryNeeds.includes('Food Allergy') || this.props.data.dietaryNeeds.includes('Other')) &&
```

Comment:
I think we should change the text for `dietaryNeedsDetails` to ask for details about your 'dietary needs', because right now it says 'food allergy' and not everyone filling it out has a food allergy.

Reply:
I see you've made this change but does this specific line also need to be changed or no?

Reply:
For this we only need to change the string, the code is okay